### ISSUE TITLE
Empty person modal looks sad

### ISSUE THREAD

**Issue by macobo** [PostHog/posthog#5600]
## Bug description

![image](https://user-images.githubusercontent.com/148820/129593279-0fd8089b-c55b-4db7-a0d0-a4f7c717e80d.png)

This happens when opening person modal on funnel historic trends.

## Expected behavior

Either:
1. Don't open a modal
2. Empty state

Also no scrollbars!

## How to reproduce

1. Construct a funnel
2. View it in historic trends
3. Click in a 0-item data point.

## Environment

- [ ] PostHog Cloud
- [ ] self-hosted PostHog, version/commit: _please provide_

## Additional context

cc @paolodamico which option would you recommend?

**Comment by clarkus** [comment]
I'm not sure a modal is needed here. The drill in action for the chart is to "inspect users" (note this should probably be persons, not users). 

<img width="438" alt="Screen Shot 2021-08-17 at 11 44 11 AM" src="https://user-images.githubusercontent.com/254612/129782932-4ed8fdbf-5af2-4d0f-a799-71a2ebba7782.png">

If there are 0 persons, there isn't a list to inspect. In 0 value cases, we could change the affordance in the chart tooltip to reiterate this state. If we remove the "click to inspect users" portion and disable the event that triggers the modal for that point, it should be more clear. Thoughts?

<img width="169" alt="Screen Shot 2021-08-17 at 11 47 39 AM" src="https://user-images.githubusercontent.com/254612/129783370-cb13286f-0e3a-401e-ad7e-7dea9048bd40.png">

**Comment by macobo** [comment]
Yup, that's option 1 under expected behaviour above

**Comment by clarkus** [comment]
> Yup, that's option 1 under expected behaviour above

Yup the core change is in the tooltip. We shouldn't prompt users to inspect users if there aren't any.
